### DISCUSSION TITLE
When running mvn clean install  get build failure on :  stack-core ......................................... FAILURE [ 15.674 s]

### DISCUSSION THREAD

**Discussion by jmbertoncelli** [eclipse-milo/milo#1141] [General]
Hello I am trying to setup the MILO project framework on my Linux UBUNTU computer but I get this issue and I do not know what to do to fix it then it would be nice if you could help me.  Thank you in advance.

[INFO] Running TestSuite
[main] WARN org.eclipse.milo.opcua.stack.core.util.validation.CertificateValidationUtil - Failed to add custom revocation checker; REVOCATION_LIST_FOUND check will be ignored.
[main] WARN org.eclipse.milo.opcua.stack.core.util.validation.CertificateValidationUtil - Failed to add custom revocation checker; REVOCATION_LIST_FOUND check will be ignored.
[milo-nonce-util-secure-random] INFO org.eclipse.milo.opcua.stack.core.util.NonceUtil - SecureRandom seeded in 0ms.
[main] INFO org.eclipse.milo.opcua.stack.core.security.DefaultTrustListManagerTest - using /tmp/securityTest
[ERROR] Tests run: 194, Failures: 2, Errors: 0, Skipped: 0, Time elapsed: 4.69 s <<< FAILURE! - in TestSuite
[ERROR] org.eclipse.milo.opcua.stack.core.util.validation.CertificateValidationUtilTest.testBuildAndValidate_LeafIntermediateSigned_Revoked  Time elapsed: 0.034 s  <<< FAILURE!
java.lang.AssertionError: expected [StatusCode{name=Bad_CertificateRevoked, value=0x801D0000, quality=bad}] but found [StatusCode{name=Bad_CertificateIssuerRevocationUnknown, value=0x801C0000, quality=bad}]
        at org.eclipse.milo.opcua.stack.core.util.validation.CertificateValidationUtilTest.testBuildAndValidate_LeafIntermediateSigned_Revoked(CertificateValidationUtilTest.java:232)

[ERROR] org.eclipse.milo.opcua.stack.core.util.validation.CertificateValidationUtilTest.testUriWithSpaces  Time elapsed: 0 s  <<< FAILURE!
org.eclipse.milo.opcua.stack.core.UaException: java.security.cert.CertificateParsingException: invalid URI name:this URI has spaces
        at org.eclipse.milo.opcua.stack.core.util.validation.CertificateValidationUtilTest.testUriWithSpaces(CertificateValidationUtilTest.java:410)
Caused by: java.security.cert.CertificateParsingException: invalid URI name:this URI has spaces
        at org.eclipse.milo.opcua.stack.core.util.validation.CertificateValidationUtilTest.testUriWithSpaces(CertificateValidationUtilTest.java:410)

[INFO] 
[INFO] Results:
[INFO] 
[ERROR] Failures: 
[ERROR]   CertificateValidationUtilTest.testBuildAndValidate_LeafIntermediateSigned_Revoked:232 expected [StatusCode{name=Bad_CertificateRevoked, value=0x801D0000, quality=bad}] but found [StatusCode{name=Bad_CertificateIssuerRevocationUnknown, value=0x801C0000, quality=bad}]
[ERROR]   CertificateValidationUtilTest.testUriWithSpaces:410 » Ua java.security.cert.Ce...
[INFO] 
[ERROR] Tests run: 194, Failures: 2, Errors: 0, Skipped: 0

**Comment by kevinherron** [comment]
Make sure you’re compiling with JDK 8 or JDK 11. Releases are done using a JDK 8.

There’s some kind of behavior change in later JDK versions that make that test fail.
